usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -621,19 +689,19 @@ def load_file_c(submission_attributes, db_cursor, award_financial_frame):
         # Award record.
 
         # Find the award that this award transaction belongs to. If it doesn't exist, create it.
-        created, award = Award.get_or_create_summary_award(
+        created, award = get_or_create_summary_award(
             awarding_agency=row['awarding_agency'],
             piid=row.get('piid'),
             fain=row.get('fain'),
             uri=row.get('uri'),
-            parent_award_id=row.get('parent_award_id'),
-            use_cache=False)
+            parent_award_id=row.get('parent_award_id'))
 
         awards_touched += [award]
 
         award_financial_data = FinancialAccountsByAwards()
 
         value_map_faba = {
+            'award': award,
```

Comment:
change award to  `award if created else None`

Reply:
Since all checks moved into `get_or_create_summary_award` that will handle returning `None` if it couldn't find an award so this check isn't necessary.